dev/packages/alpha/cisco-0.0.2/dataset/asa/elasticsearch/ingest-pipeline/default.yml

```diff
@@ -0,0 +1,1281 @@
+---
```

Comment:
If this file contains any fields not mentioned in the fields files (e.g. event.code, event.provider), please create in Beats a file called `fields.epr.yml` and describe them there (there is a paragraph about this in CONTRIBUTING).